core/controller.c

```diff
@@ -136,6 +138,48 @@ static void adapter_dump_state(int adapter)
   gstatus("\n");
 }
 
+static float float_swap(float x)
+{
+  union V {
+    float f;
+    uint32_t i;
+  };
+  union V val;
+  val.f = x;
+  val.i = htonl(val.i);
+  return val.f;
+}
+
+static void handlePacketConfig(const s_network_packet_config* buf)
+{
+  float s = float_swap(buf->sensibility);
+  if (s < FLT_MAX && s >= 0) {
+    printf("setting sensibility=%f\n", s);
```

Comment:
Avoid using printf. Maybe you can use ginfo() and only print if gimx_params.debug.controller is set?